layouts/joomla/form/field/media.php

```diff
@@ -73,12 +74,19 @@
 }
 
 // Pre fill the contents of the popover
-if ($showPreview) {
-	if ($value && file_exists(JPATH_ROOT . '/' . $value)) {
-		$src = Uri::root() . $value;
-	} else {
+if ($showPreview)
+{
+	$cleanValue = MediaHelper::getCleanMediaFieldValue($value);
+
+	if ($cleanValue && file_exists(JPATH_ROOT . '/' . $cleanValue))
+	{
+		$src = Uri::root() . $cleanValue;
```

Comment:
```suggestion
		$src = Uri::root() . $value;
```

I would prefer if we keep the original value here, eg: `$src = Uri::root() . $value;` to. match the rendered preview source from JS land. Check my images in your issue